### PR TITLE
Added the possibility to hide wielded items in-game

### PR DESCRIPTION
**Pull request by G1ov4**

### CODE REVIEW

File: wieldview/README.txt [hunk 1]
```diff
@@ -21,3 +21,7 @@ Wield image transformation: To apply a simple transformation to the item in
 hand, add the group “wieldview_transform” to the item definition. The group
 rating equals one of the numbers used for the [transform texture modifier
 of the Lua API.
+
+Disabling the feature in-game: If you want to hide the wielded item
+you can add an INT metadata to the player called "show_wielded_item" and set
+it to 2 (any other value will show the wielded item again)
```
Comment (Panquesito7) [inline wieldview/README.txt:27]:
```suggestion
it to 2 (any other value will show the wielded item again).
```

Comment (Panquesito7) [reply wieldview/README.txt:27]: Not resolved yet.

Comment (BuckarooBanzay) [reply wieldview/README.txt]: missing punctuations, don't we have other problems? :smirk:

**Comment by BuckarooBanzay** [comment]
thank you for your contribution :+1:

**Rant:** The `player:set_meta()` approach is a bit weird for other mods to use but i guess this can be incorporated in a later stage to something like `wieldview:disable_wieldview(player)` and `wieldview:enable_wieldview(player)` which in turn set the metadata...

I'm curios: whats your use-case / mod in which you are using that?

**Comment by G1ov4** [comment]
My friends and I are making a minigames server, and for some of them we needed to hide the wielded item (e.g. murder mod, if anyone could see the wielded item it would be easy to discover the murderer), that's why I needed it.

**Pull request merged**